CRM/Contribute/Form/AbstractEditPayment.php

```diff
@@ -213,6 +213,7 @@ public function preProcess() {
     $this->assign('contactID', $this->_contactID);
     CRM_Core_Resources::singleton()->addVars('coreForm', array('contact_id' => (int) $this->_contactID));
     $this->_action = CRM_Utils_Request::retrieve('action', 'String', $this, FALSE, 'add');
+    $this->_mode = empty($this->_mode) ? CRM_Utils_Request::retrieve('mode', 'String', $this) : $this->_mode;
```

Comment:
@eileenmcnaughton I have changed this assignment, as earlier there was a test build failure https://test.civicrm.org/job/CiviCRM-Core-PR/14781/testReport/junit/(root)/CRM_Member_Form_MembershipRenewalTest/testSubmit/  due to override ```_mode``` variable override in Membership backoffice form.